### ISSUE TITLE
Failed to install mcpi modded

### ISSUE THREAD

**Issue by Parindraa** [Botspot/pi-apps#1666]
### Confirmations

- [X] I searched for similar bug reports in https://github.com/Botspot/pi-apps/issues and found none were relevant.
- [X] I tried updating Pi Apps and the issue is still not fixed. [How to update Pi Apps?](https://github.com/Botspot/pi-apps#to-update-pi-apps)

### What happened?

Keeps saying APT reported a unsolvable repository, check your internet connection.

### Description

Maybe it doesn't support arm64, that is the reason why it happens.

### What is your operating system?

Raspberry pi Os Arm64

### (Optional) Error log? Terminal output? Debug messages?

```bash
deb [trusted=yes] https://mcpi-revival.github.io/mcpi-packages/ bullseye main
1) gMCPIL (recommended, unmaintained)	3) Planet (Unofficial, but maintained)
2) jMCPIL (better GUI, unmaintained)	4) Quit
Which launcher for MCPI Modded would you like to use? Need help deciding? Ask here: https://discord.com/invite/aDqejQGMMy
1
Will install these packages: gmcpil python3-pip python3-dev 
Waiting until APT locks are released... Done
Running sudo apt update...
Hit:1 http://security.debian.org/debian-security bullseye-security InRelease
Hit:2 http://deb.debian.org/debian bullseye InRelease
Hit:3 http://deb.debian.org/debian bullseye-updates InRelease
Ign:4 https://mcpi-revival.github.io/mcpi-packages bullseye InRelease
Ign:5 https://mcpi-revival.github.io/mcpi-packages bullseye Release
Ign:6 https://mcpi-revival.github.io/mcpi-packages bullseye/main all Packages
Ign:7 https://mcpi-revival.github.io/mcpi-packages bullseye/main armhf Packages
Hit:8 http://archive.raspberrypi.org/debian bullseye InRelease
Ign:9 https://mcpi-revival.github.io/mcpi-packages bullseye/main arm64 Packages
Ign:10 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US.UTF-8
Ign:11 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US
Ign:12 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en
Ign:6 https://mcpi-revival.github.io/mcpi-packages bullseye/main all Packages
Ign:7 https://mcpi-revival.github.io/mcpi-packages bullseye/main armhf Packages
Ign:9 https://mcpi-revival.github.io/mcpi-packages bullseye/main arm64 Packages
Ign:10 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US.UTF-8
Ign:11 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US
Ign:12 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en
Ign:6 https://mcpi-revival.github.io/mcpi-packages bullseye/main all Packages
Ign:7 https://mcpi-revival.github.io/mcpi-packages bullseye/main armhf Packages
Ign:9 https://mcpi-revival.github.io/mcpi-packages bullseye/main arm64 Packages
Ign:10 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US.UTF-8
Ign:11 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US
Ign:12 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en
Ign:6 https://mcpi-revival.github.io/mcpi-packages bullseye/main all Packages
Ign:7 https://mcpi-revival.github.io/mcpi-packages bullseye/main armhf Packages
Ign:9 https://mcpi-revival.github.io/mcpi-packages bullseye/main arm64 Packages
Ign:10 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US.UTF-8
Ign:11 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US
Ign:12 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en
Ign:6 https://mcpi-revival.github.io/mcpi-packages bullseye/main all Packages
Ign:7 https://mcpi-revival.github.io/mcpi-packages bullseye/main armhf Packages
Ign:9 https://mcpi-revival.github.io/mcpi-packages bullseye/main arm64 Packages
Ign:10 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US.UTF-8
Ign:11 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US
Ign:12 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en
Ign:6 https://mcpi-revival.github.io/mcpi-packages bullseye/main all Packages
Ign:7 https://mcpi-revival.github.io/mcpi-packages bullseye/main armhf Packages
Ign:9 https://mcpi-revival.github.io/mcpi-packages bullseye/main arm64 Packages
Ign:10 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US.UTF-8
Ign:11 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US
Ign:12 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en
Ign:6 https://mcpi-revival.github.io/mcpi-packages bullseye/main all Packages
Err:7 https://mcpi-revival.github.io/mcpi-packages bullseye/main armhf Packages
  404  Not Found [IP: 185.199.111.153 443]
Ign:9 https://mcpi-revival.github.io/mcpi-packages bullseye/main arm64 Packages
Ign:10 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US.UTF-8
Ign:11 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US
Ign:12 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en
E: Failed to fetch https://mcpi-revival.github.io/mcpi-packages/dists/bullseye/main/binary-armhf/Packages  404  Not Found [IP: 185.199.111.153 443]
E: Some index files failed to download. They have been ignored, or old ones used instead.
apt update complete.
Failed to run sudo apt update!
APT reported these errors:
E: Failed to fetch https://mcpi-revival.github.io/mcpi-packages/dists/bullseye/main/binary-armhf/Packages  404  Not Found [IP: 185.199.111.153 443]
E: Some index files failed to download. They have been ignored, or old ones used instead.
Hit:1 http://security.debian.org/debian-security bullseye-security InRelease
Hit:2 http://deb.debian.org/debian bullseye InRelease
Hit:3 http://deb.debian.org/debian bullseye-updates InRelease
Ign:4 https://mcpi-revival.github.io/mcpi-packages bullseye InRelease
Ign:5 https://mcpi-revival.github.io/mcpi-packages bullseye Release
Ign:6 https://mcpi-revival.github.io/mcpi-packages bullseye/main all Packages
Ign:7 https://mcpi-revival.github.io/mcpi-packages bullseye/main armhf Packages
Hit:8 http://archive.raspberrypi.org/debian bullseye InRelease
Ign:9 https://mcpi-revival.github.io/mcpi-packages bullseye/main arm64 Packages
Ign:10 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US.UTF-8
Ign:11 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US
Ign:12 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en
Ign:6 https://mcpi-revival.github.io/mcpi-packages bullseye/main all Packages
Ign:7 https://mcpi-revival.github.io/mcpi-packages bullseye/main armhf Packages
Ign:9 https://mcpi-revival.github.io/mcpi-packages bullseye/main arm64 Packages
Ign:10 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US.UTF-8
Ign:11 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US
Ign:12 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en
Ign:6 https://mcpi-revival.github.io/mcpi-packages bullseye/main all Packages
Ign:7 https://mcpi-revival.github.io/mcpi-packages bullseye/main armhf Packages
Ign:9 https://mcpi-revival.github.io/mcpi-packages bullseye/main arm64 Packages
Ign:10 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US.UTF-8
Ign:11 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US
Ign:12 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en
Ign:6 https://mcpi-revival.github.io/mcpi-packages bullseye/main all Packages
Ign:7 https://mcpi-revival.github.io/mcpi-packages bullseye/main armhf Packages
Ign:9 https://mcpi-revival.github.io/mcpi-packages bullseye/main arm64 Packages
Ign:10 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US.UTF-8
Ign:11 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US
Ign:12 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en
Ign:6 https://mcpi-revival.github.io/mcpi-packages bullseye/main all Packages
Ign:7 https://mcpi-revival.github.io/mcpi-packages bullseye/main armhf Packages
Ign:9 https://mcpi-revival.github.io/mcpi-packages bullseye/main arm64 Packages
Ign:10 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US.UTF-8
Ign:11 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US
Ign:12 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en
Ign:6 https://mcpi-revival.github.io/mcpi-packages bullseye/main all Packages
Ign:7 https://mcpi-revival.github.io/mcpi-packages bullseye/main armhf Packages
Ign:9 https://mcpi-revival.github.io/mcpi-packages bullseye/main arm64 Packages
Ign:10 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US.UTF-8
Ign:11 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US
Ign:12 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en
Ign:6 https://mcpi-revival.github.io/mcpi-packages bullseye/main all Packages
Err:7 https://mcpi-revival.github.io/mcpi-packages bullseye/main armhf Packages
  404  Not Found [IP: 185.199.111.153 443]
Ign:9 https://mcpi-revival.github.io/mcpi-packages bullseye/main arm64 Packages
Ign:10 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US.UTF-8
Ign:11 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en_US
Ign:12 https://mcpi-revival.github.io/mcpi-packages bullseye/main Translation-en
E: Failed to fetch https://mcpi-revival.github.io/mcpi-packages/dists/bullseye/main/binary-armhf/Packages  404  Not Found [IP: 185.199.111.153 443]
E: Some index files failed to download. They have been ignored, or old ones used instead.
```

**Comment by Rak1ta** [comment]
I messaged them on discord about it.
https://discord.gg/vGMdfv2T

**Comment by nianob** [comment]
Same for me.
I'd love to see an solution for this

**Comment by theofficialgman** [comment]
the repo was discontinued without any warning, this has immediately broken 1000s of users installs
<img width="553" alt="image" src="https://user-images.githubusercontent.com/28281419/162045429-ff81b92c-cda5-4899-bce5-abc64fc5a2ba.png">

**Comment by theofficialgman** [comment]
@Parindraa @nianob please try installing again. the apt repo has been temporarily re-enabled

future updates will come via appimages but those have not been implemented yet

**Comment by Botspot** [comment]
Closing, as this issue is currently solved and a complete fix is in the works at #1673.
